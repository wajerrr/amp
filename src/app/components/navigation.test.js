import React from 'react';
import renderer from 'react-test-renderer';
import Navigation from './navigation';

describe('Navigation', () => {
  it('should match snapshot', () => {
    const tree = renderer.create(<Navigation />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
