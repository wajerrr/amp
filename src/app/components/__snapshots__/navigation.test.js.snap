// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navigation should match snapshot 1`] = `
<nav
  className="css-t6plsr"
>
  <a
    className="css-1l3b7az"
    href="/"
    title="The Economist"
  >
    <span
      className="css-38lglc"
    >
      The Economist
    </span>
  </a>
  <a
    className="css-vwbibu"
    href="https://econ.st/2bs1gXz"
    role="button"
  >
    Subscribe
  </a>
  <button
    className="css-1eqchrp"
    on="tap:sidebar.toggle"
    tabIndex="0"
  />
</nav>
`;
